web-app/src/utilities/menuIcons.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
I think I get _why_ you've made this file, but it seems like an abstraction that we won't benefit much from because there's only ever going to be 1 component needing this data. I'd just use the <FontAwesomeIcon /> component directly in the menu.